docs/spelling_wordlist

```diff
@@ -19,6 +19,7 @@ Ctrl
 dialogs
 Django
 draggable
+filepath
```

Comment:
Not sure this should be added to the dictionary; "file path" would be preferable usage.

Reply:
I had thought that the compound form was, at least in programming contexts, the more common one. But yeah, looking it up, I see usage is much more divided than I realized. Reverted.